docs/Governance.md

```diff
@@ -0,0 +1,223 @@
+
+### The Lead Maintainer Position & The Holder Of That Position
+
+At least once a year, provisionally September 14th, the PLC is mandated to
+affirm or withhold its consent for the lead maintainer role to exist in its
+current form as a distinct entity. The lead maintainer may vote on this issue.
+In the event of a tie regular maintainers are asked to vote on the issue and
+collectively act as the sole tie-breaking vote.
+
+On the same day the PLC is additionally mandated to vote on whether to affirm
+or withhold its consent for the current occupant of the lead maintainer role to
+remain in post. The lead maintainer _may not_ participate in this vote, and
+again in the event of a tie regular maintainers will be asked to provide a
+collective tiebreaker.
+
+Neither of the above bar any member of the PLC calling a vote on either issue
+at an earlier point in time, although doing so should be considered frowned
+upon in all but the most severe of circumstances. In both votes a simple
+majority suffices as a valid determination of the will of the PLC.
+
+If the lead maintainer position becomes vacant for any reason the PLC is
+mandated to fill the vacancy, unless a vote is held to absorb the position.
+Any maintainer or member of the PLC can nominate a new lead maintainer, but
+confirming a new lead maintainer requires every member of the PLC to vote
+and 2/3 of those votes to be in the affirmative.
+
+### Maintainer Pull Requests and Issues
+
+Maintainers may not close other maintainer's Pull Requests or Issues without
+the consent of the maintainer in question. The two exceptions of this rule are:
+
+_Inactivity_: If a Pull Request or Issue is inactive for 28 days, any maintainer
+is welcome to nudge the maintainer who opened the PR/Issue for an update on
+where things stand, and if one is not provided within two days the issue can be
+closed by any maintainer. If the maintainer who opened the PR/Issue wishes to
+return to it later they are welcome to reopen the PR/Issue.
+
+_Simple Changes_: If a maintainer submits, for example, a minor documentation
+tweak to `homebrew/brew` or a simple version bump to `homebrew/core` any
+maintainer is welcome to merge that in without waiting for the original
+maintainer to do so. If maintainers object to this they should add the
+`do not merge` label to the PR/Issue. That label may be ignored if the issue
+is an urgent security fix or critical to the continued function of Homebrew's
+CI.
+
+Maintainers should not routinely revert the work of other maintainers without
+first discussing what the issue is with that maintainer. However, if the
+problem is significant and the original maintainer cannot be reached within an
+hour any maintainer may revert a problematic commit. It is recommended to ping
+the original maintainer into any revert PR so they can comment later if they
+wish to, and so they're aware of the revert without having to go fishing in the
+commit log.
+
+### Maintainer Conduct
```

Comment:
This whole section uses `&` instead of `and`. Is that intentional?

Reply:
Not a conscious choice. I can change it it people prefer `and` throughout; I just tend to lean on `&` to condense things a little.

Reply:
I prefer `and`

Reply:
Will change 👍.